website/src/contribute/release-guide.md

```diff
@@ -769,13 +815,15 @@ Template:
 1. Maven artifacts deployed to the staging repository of [repository.apache.org](https://repository.apache.org/content/repositories/)
 1. Source distribution deployed to the dev repository of [dist.apache.org](https://dist.apache.org/repos/dist/dev/beam/)
 1. Website pull request proposed to list the [release]({{ site.baseurl }}/get-started/downloads/), publish the [Java API reference manual](https://beam.apache.org/releases/javadoc/), and publish the [Python API reference manual](https://beam.apache.org/releases/pydoc/).
+1. Docker images are published to [DockerHub](https://hub.docker.com/search?q=apachebeam&type=image) with tags :{RELEASE}_rc.
```

Comment:
with tags: `{RELEASE}_rc`.
Just realized there is a `:` in the tag. If that's the standard way to express a tag, please add it back.